8/README.MD

```diff
@@ -6,7 +6,7 @@
 
 Python is a much newer language than C. C was first released in 1972, Python in 1991.
 
-Python is an interpreted language. This means that the code never gets compiled, but rather an interpreter translates the code to machine code as it gets executed.
```

Comment:
Technically, Python is compiled into byte code (hence the .pyc files), and then that byte code is interpreted line by line; it's not converted to machine code per se. Best not to call any language "compiled" or "interpreted" since any can be either, or at least add a "usually" disclaimer. Just so happens that C is usually compiled, but it could be interpreted too. Better to say, e.g., "Unlike a C program, which typically has to be compiled before you can run it, a Python program can be run without explicitly compiling it first." Or some such nuance.